### ISSUE TITLE
Could I use `res.render` to render a component

### ISSUE THREAD

**Issue by briefguo** [kyle-mccarthy/nest-next#6]
Hello

Use `res.render`, I could do this
```ts
...
  @Get()
  // @Render('User/LoginForm')
  public async login(@Res() res: any) {
    return res.render('User/LoginForm');
  }
...
```
```ts
...
  @Get()
  public async login(@Res() res: any) {
    if (condition A) {
       return res.render('A');
    }
    return res.render('B');
  }
...
```

I think that `nest-next` it run as a  engine powered by `next`, and next provides a few of some render method. Is there some samples to use these `renderMethod` easily?

![image](https://user-images.githubusercontent.com/9976575/52908148-e63b8400-32aa-11e9-9107-5d13328ee786.png)

**Comment by kyle-mccarthy** [comment]
Hi I just checked and as of right now, res.render does not correctly reference the next render method. Let me look into this and see what I need to do to make that possible.

**Comment by kyle-mccarthy** [comment]
@briefguo I released a new version (v0.4.0) which exposes the render method on the response object. You can use it like the example you proposed above. Additionally it takes an object in the second parameter allowing for data to be passed to the page.

```
interface ServerResponse {
  render: (view: string, data?: any) => any;
}

...
@Get()
public login(@Res() res: ServerResponse) {
  ...
  return res.render('LoginForm', { errors: ['Invalid email or password'] });
}
...

```

**Comment by briefguo** [comment]
@kyle-mccarthy  Thanks, it's very useful for me
